### PR TITLE
complete task (objects 12597)

### PR DESCRIPTION
**Pull request by Irina-N23**
Task #12597 has been completed.
Just check whether everything is correct in it!

### CODE REVIEW

File: objects/objects12597.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+// Напишите код для суммирования всех зарплат и сохраните результат в переменной sum. Должно получиться 390.
+// Если объект salaries пуст, то результат должен быть 0.
+
+let salaries = {
+    John: 100,
+    Ann: 160,
+    Pete: 130
+}
+
+let countTotalSalary = (salariesInfo) => {
```
Comment (SiarheiRylach) [inline objects/objects12597.js]: const

File: objects/objects12597.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+// 4. У нас есть объект, в котором хранятся зарплаты нашей команды:
+// let salaries = {
+//  John: 100,
+//  Ann: 160,
+//  Pete: 130
+// }
+// Напишите код для суммирования всех зарплат и сохраните результат в переменной sum. Должно получиться 390.
+// Если объект salaries пуст, то результат должен быть 0.
+
+let salaries = {
```
Comment (SiarheiRylach) [inline objects/objects12597.js]: conts

File: objects/objects12597.js (hunk 1 above)
Comment (Irina-N23) [reply objects/objects12597.js]: Thank you! Fixed.

File: objects/objects12597.js (hunk 2 above)
Comment (Irina-N23) [reply objects/objects12597.js]: Thank you! Fixed.

**Pull request merged**